app/javascript/components/vm-edit-form/vm-edit-form.schema.js

```diff
@@ -55,6 +55,10 @@ function createSchema(emsId, parentOptions) {
           moveLeftTitle: __('Move selected VMs to left'),
           moveRightTitle: __('Move selected VMs to right'),
           moveAllRightTitle: __('Move all VMs to right'),
+          moveAllLeftTitle: __('Remove All'),
```

Comment:
Can we also mark the "Filter values" for translation or is there no prop for that?

Reply:
see https://github.com/ManageIQ/manageiq-ui-classic/pull/8282#discussion_r885620027